### ISSUE TITLE
Fit minimumScaleMode does not work properly when the map is rotated

### ISSUE THREAD

**Issue by Nohus** [peterLaurence/MapCompose#11]
With `minimumScaleMode = Fit` and the map rotated 45°, it is impossible to zoom out enough to see the entire map. Instead the corners of the map are offscreen.

**Comment by peterLaurence** [comment]
Yes, I'm aware of that. For now, it hasn't been a big issue (for me).
Fixing this requires having the minimum scale depend on the rotation.

**Comment by p-lr** [comment]
The new 2.4.0 version has proper support for `setScrollOffsetRatio`. When using it with e.g 0.3f value, it's possible to scroll enough to see the edges.
